files-to-sync/cli/commands/destroy.ts

```diff
@@ -0,0 +1,107 @@
+// This file is managed by macpro-mdct-core so if you'd like to change it let's do it there
+import { Argv } from "yargs";
+import {
+  CloudFormationClient,
+  DeleteStackCommand,
+  waitUntilStackDeleteComplete,
+} from "@aws-sdk/client-cloudformation";
+import { checkIfAuthenticated } from "../lib/sts.js";
```

Comment:
Have you confirmed that all of the apps have these things (checkIfAuthenticated, project, region, delete_topics) available for import? 

NOT in this pr but at some point I'd like to standardize the casing for naming functions (delete_topics should be deleteTopics imho).

Reply:
I know some do not have delete_topics and download clam av.  I plan on manually adding  dummy versions of them to the PR that gets created by this process.